tests/features/collection/og.feature

```diff
@@ -75,10 +81,11 @@ Feature: Organic Groups integration
     And I should see the "Join this collection" button
     And the "Überwaldean Land Eels" collection should have 0 members
 
-    When I go to the homepage of the "Folk Dance and Song Society" collection
+    When I am logged in as "Kathie Cumberwatch"
+    And I go to the homepage of the "Folk Dance and Song Society" collection
```

Comment:
The workarounds that are added in this test are indicating that we have missing functionality. The test literally says "Check that the users can leave the collections" but now it is no longer possible to leave a collection if your membership is pending.

The workarounds that are being added in this test are making the test less focused. The test is about joining and leaving through the user interface, but now we have users that are created through the API, and users that are not leaving.

I would try to keep the spirit of the test intact, drop the additional user that was added, and instead approve their membership so they can still leave the collection. We also need a followup so users can cancel their pending membership, and add documentation here why were are jumping through hoops.